conf/application.conf

```diff
@@ -164,35 +164,66 @@ versioning {
 api-config {
 
   scopes {
-    "dummy-scope" {
-      fields = [ "A","B","C","D","E","F","G","H","I","J","K","L" ]
+    "read:individuals-benefits-and-credits-laa-c1" {
+        fields = ["B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
     }
-    "read:hello-scopes-1" {
-      fields = [ "A","B","C"]
+    "read:individuals-benefits-and-credits-laa-c2" {
+        fields = ["B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
     }
-    "read:hello-scopes-2" {
-      fields = [ "A","B"]
+    "read:individuals-benefits-and-credits-laa-c3" {
+        fields = ["B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
+    }
+    "read:individuals-benefits-and-credits-hmcts-c2" {
+        fields = ["A", "B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
+    }
+    "read:individuals-benefits-and-credits-hmcts-c3" {
+        fields = ["A", "B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
+    }
+    "read:individuals-benefits-and-credits-lsani-c1" {
+        fields = ["A", "B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
+    }
+    "read:individuals-benefits-and-credits-lsani-c3" {
+        fields = ["A", "B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
     }
   }
 
   endpoints {
-    paye {
-      endpoint = "/individuals/employments/paye?matchId=$randomMatchId{&fromDate,toDate}"
-      fields {
-        A = "payments"
-        B = "employer/employerName"
-        C = "employer/employerAddress/line1"
-        D = "employer/employerAddress/line2"
-        E = "employer/employerAddress/line3"
-        F = "employer/employerDistrictNumber"
-        G = "employer/employerSchemeReference"
-        H = "employmentStartDate"
-        I = "employmentLeavingDate"
-        J = "employmentPayFrequency"
-        K = "employeeAddress"
-        L = "payrollId"
+    benefitsAndCredits {
+        endpoint = "individuals/benefits-and-credits"
```

Comment:
Update to correct HAL link and ensure endpoint keys match those in the controllers `getEndPointScopes` invocation.